app/mathtoolbar.cpp

```diff
@@ -30,6 +30,9 @@ MathToolbar::MathToolbar(QWidget* parent) : QToolBar(parent) {
     spacer->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Preferred);
     addWidget(spacer);
 
+    addAction(new SettingsAction("Š", "\\settings", this));

```

Comment:
The keyword tooltips should be updated dynamically if the keywords are allowed to change. E.g. have a function `std::string getKeyword(const std::string& result);`. Probably include a description and have the keyword in parenthesis.

Reply:
This is tangential to the current PR. Do it later in [FOR-114](https://forscape.atlassian.net/browse/FOR-114)